src/character.cpp

```diff
@@ -1957,3 +1957,11 @@ bool Character::made_of( const material_id &m ) const {
     static const std::vector<material_id> fleshy = { material_id( "flesh" ), material_id( "hflesh" ) };
     return std::find( fleshy.begin(), fleshy.end(), m ) != fleshy.end();
 }
+
+bool Character::is_blind() const
+{
+    if( worn_with_flag( "BLIND" ) || has_effect( effect_blind ) ) {
```

Comment:
@codemime I see some conditions where ```has_active_bionic("bio_blindfold")``` isn't checked or checked for the inverted state, so I think it's good as it is.